### ISSUE TITLE
Can you add an option to randomly flip cards and another idea?

### ISSUE THREAD

**Issue by Madmaxneo** [alexkurowski/solo-toolkit#6]
This is mainly for the Oracle cards. When a card is drawn and it's upside down it usually means the opposite of what the card means. This would make it much more viable for games that use Oracle cards.

The other idea I have is the ability to create custom layouts for the cards. I have a few different card decks that are made for RPGs (like the Oracle Character Generator, Deck of Worlds, and a few others). I have images for all the cards and it would be really cool if I could set up custom layouts for how the cards appear or where I can place them so that they are more in line with how the different systems work that use them.

**Comment by alexkurowski** [comment]
Just released 0.3.6 version that adds a 50% chance of a flipped card 🙌 
Custom decks can't be flipped for now 😓  But maybe I could add a configuration file for custom decks, not sure how it would look like.

I'm not exactly clear on what you mean by custom layouts. It does make sense to have a virtual table where you can arrange your cards though, I agree!
In 0.3.6 I added an option for copying images that you draw from a deck, so you can click on a card and ctrl+V it into a note or a canvas. The feature is a bit trash right now because it'll create new "Pasted image" file in your New attachments folder every time.
I'll try to make it reference the correct file instead a bit later.

**Comment by alexkurowski** [comment]
Oh, that's interesting, I haven't heard about it. Will definitely look into it more.

I've released 0.3.7 just now. With it, if you create a note `config` in the same deck folder, and add a line in it `flip4`. After refresh it will rotate images at random in all four directions.
Let me know if it helps!

**Comment by Madmaxneo** [comment]
Yeah the deck of worlds cards have 4 variable sides but if you figure out
the copy and paste thing I can rotate them all I want in canvas or
excalidraw.
It'll be great once you figure out the flip thing for custom cards but
having the option to turn it off and on is important also.
One deck of cards I use has negative cards for each of the major arcana
cards so those don't need to be flipped.

**Comment by alexkurowski** [comment]
A new option has been added in version 0.3.12!

Unfortunately, I can't think of a good way to keep the flip info without creating unnecessary image duplicates. But if you pick "File path" card click option in plugin settings, you should be able to immediately paste the card into Excalidraw and rotate it manually.

**Comment by Madmaxneo** [comment]
> Obsidian Image Toolkit allows you to "pin" a rotated image to the screen

But will it create a new pasted image in the files?

**Comment by alexkurowski** [comment]
I played around with Image Toolkit, it don't create duplicates but it only allows you to pin an image on the screen and it don't seem to persist between sessions.

I think for something like Deck of Worlds it's best to pick "File path" option, copy-paste a card into Excalidraw and arrange it manually.

**Comment by RodWarrior** [comment]
> Just released 0.3.6 version that adds a 50% chance of a flipped card 🙌 Custom decks can't be flipped for now 😓 But maybe I could add a configuration file for custom decks, not sure how it would look like.
> 
> I'm not exactly clear on what you mean by custom layouts. It does make sense to have a virtual table where you can arrange your cards though, I agree! In 0.3.6 I added an optional feature for copying images that you draw from a deck, so you can click on a card and ctrl+V it into a note or a canvas. The feature is a bit trash right now because it'll create new "Pasted image" file in your New attachments folder every time. I'll try to make it reference the correct file instead a bit later.

Hello @alexkurowski ! I would like to ask to the flipping of the deck of cards to be optional by selecting an option o the menu. Most ways i use the tarot don't use it flipped, and it changes the meaning of the pull. I love having the option, but would like it to be optional, not default behaviour (or at least that it can be a choice). Thank you for your hard work, is realy amazing to see how much you update the plugin and listen to feedback, I really appreciate it!

**Comment by alexkurowski** [comment]
Good point, added an option in 0.3.19.

**Comment by RodWarrior** [comment]
> Good point, added an option in 0.3.19.

Thanks, you're the GOAT
